knowledge-base/binaryformatter-exception-after-upgrade-to-net5.md

```diff
@@ -0,0 +1,48 @@
+---
+title: BinaryFormatter serialization and deserialization are disabled within this application is thrown in .NET5
+description: BinaryFormatter serialization and deserialization are disabled within this application after upgrade to .NET5
+type: troubleshooting
+page_title: BinaryFormatter serialization and deserialization are disabled within this application is thrown in .NET5
+slug: binaryformatter-exception-after-upgrade-to-net5
+position: 
+tags: .NET 5.0
+ticketid: 1494918
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
```

Comment:
You may include the version here, for example, 'up to R3 2020', because the KB will probably not concern the new releases, where we will support .NET5 out of the box.